core/src/actions/status.ts

```diff
@@ -18,6 +18,7 @@ export class PublicStatus extends OptionallyAuthenticatedAction {
 
   async runWithinTransaction({ response }) {
     response.status = "ok";
+    return response;
   }
```

Comment:
I think this is from an old version of actionhero.  Best to just `return {status: 'ok'}`

Reply:
I think I ran into an issue with TypeScript on this one. I'll give it a shot.